src/assets/scripts/client/InputController.js

```diff
@@ -141,10 +143,25 @@ export default class InputController {
     }
 
     /**
+     * Enable handlers
+     *
      * @for InputController
      * @method setupHandlers
      */
     setupHandlers() {
+        this.$body.addEventListener('contextmenu', (event) => this._disableRightClickMenu(event));
+
+        return this;
+    }
+
+    /**
+     * Disable handlers
+     *
+     * @for InputController
+     * @method setupHandlers
+     */
+    disableHandlers() {
+        this.$body.removeEventListener('contextmenu', (event) => this._disableRightClickMenu(event));
```

Comment:
this method can be removed with the contents moved to `.disable()`

Reply:
done